Dark Theme/DarkTheme.js

```diff
@@ -73,15 +78,38 @@ tau
             addCSSRule('.tau-board-view .tau-label__velocity, .tau-board-view .tau-label__effort', 'color: #A1A7B3 !important;');
             addCSSRule('.tau-paging-info', 'color: #fff !important; font-weight: 400 !important');
             addCSSRule('.boardsettings-filter__contener:after', 'background: #292c33 !important; box-shadow: none !important;');
-            addCSSRule('.tau-cellholder .tau-label:after, .tau-x-header .tau-label:after, .tau-backlog-header .tau-label:after', 'box-shadow: none !important;')
-        };
+            addCSSRule('.tau-cellholder .tau-label:after, .tau-x-header .tau-label:after, .tau-backlog-header .tau-label:after', 'box-shadow: none !important;');
+        }};
         topMenu.addItem('TV').onClick(function() {
             var $st = $(document).find('#tv-style');
+            var d = new Date();
+            d.setTime(d.getTime() + (365*24*60*60*1000));
+            var expires = "expires="+ d.toUTCString();
             if ($st.length) {
+                document.cookie = "TPtheme=Light; " + expires;
                 $st.remove();
+                
             } else {
+                document.cookie = "TPtheme=Dark; " + expires;
                 init();
+                
             }
         });
+        function getCookie(cname) {
+            var name = cname + "=";
```

Comment:
Please, rewrite with bit of functional programming ([filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)) for easy reading:

```
var name = cname + "=";
var themeCookies = document.cookie.split(';').filter(function(cookie) {
    return cookie.indexOf(name) >= 0;
});

// No such cookie found.
if (themeCookies.length === 0) return '';

// Cookie has format key=value.
return themeCookies[0].split('=')[1];
```